src/main/java/com/delphix/masking/initializer/maskingApi/SetupDriver.java

```diff
@@ -282,19 +317,23 @@ private void handleFileFieldMetadata(List<FileFieldMetadata> fileFieldMetadatas,
 
         for (FileFieldMetadata fileFieldMetadata : fileFieldMetadatas) {
 
-            Integer id = getFileFieldMetadatas
-                    .getFileFieldMetadatas()
-                    .stream()
-                    .filter(f -> f.getFileFormatId().equals(fileFormatId) && f.getFieldName().equals
-                            (fileFieldMetadata.getFieldName()))
-                    .findFirst()
-                    .orElseThrow(RuntimeException::new)
-                    .getFileFieldMetadataId();
+            try {
+                Integer id = getFileFieldMetadatas
+                        .getFileFieldMetadatas()
+                        .stream()
+                        .filter(f -> f.getFileFormatId().equals(fileFormatId) && f.getFieldName().equals
+                                (fileFieldMetadata.getFieldName()))
+                        .findFirst()
+                        .orElseThrow(RuntimeException::new)
+                        .getFileFieldMetadataId();
+
+                fileFieldMetadata.setFileFieldMetadataId(id);
 
-            fileFieldMetadata.setFileFieldMetadataId(id);
+                PutFileFieldMetadata putFileFieldMetadata = new PutFileFieldMetadata(fileFieldMetadata);
+                apiCallDriver.makePutCall(putFileFieldMetadata);
+            } catch (Exception e) {
 
```

Comment:
Why are we swallowing the exception here?

Reply:
In case of multiple record types, it is causing error. So I ignored the error so that other setups can continue.

Reply:
Again, if the error is unrelated to record types, then we have falsely told the user that everything succeeded. I suggest adding another flag and logging to indicate this.

Reply:
Used the ignoreErrors flag to ignore this error conditionally.